Quasardb/Table/Reader/QdbCell.cs

```diff
@@ -95,6 +95,10 @@ public byte[] BlobValue
                     out pointer_t content, out size_t length);
                 if (err == qdb_error.qdb_e_element_not_found)
                     return null;
+                if ((int)length <= 0)
+                    return null;
+                if (content == null)
```

Comment:
We should add this check in all the places where we do `Marshal.Copy`. Adding a helper for this would be a good idea.

Reply:
Have you grepped for `Marshal.Copy`?